platform/fabric/core/generic/config/testdata/core.yaml

```diff
@@ -2,53 +2,123 @@ fabric:
   enabled: true
   default:
     default: true
-    BCCSP:
-      Default: SW
-      SW:
-        Hash: SHA2
-        Security: 256
-        FileKeyStore:
-          KeyStore:
-    mspConfigPath: /Users/adc/golang/src/github.com/hyperledger-labs/fabric-smart-client/testdata/fabric/crypto/peerOrganizations/org1.example.com/peers/approver.org1.example.com/msp
-    localMspId: Org1MSP
+    mspCacheSize: 500
     msps:
+      - id: alice
+        default: false
+        mspType: bccsp
+        mspID: Org1MSP
+        cacheSize: 0
+        path: /home/vagrant/testdata/fabric.default/crypto/peerOrganizations/org1.example.com/users/alice@org1.example.com/msp
+        opts:
+          BCCSP:
+            Default: SW
+            # Settings for the SW crypto provider (i.e. when DEFAULT: SW)
+            SW:
+              Hash: SHA2
+              Security: 256
+            # Settings for the PKCS#11 crypto provider (i.e. when DEFAULT: PKCS11)
+            PKCS11:
+              # Location of the PKCS11 module library
+              Library: /usr/local/lib/softhsm/libsofthsm2.so
+              # Token Label
+              Label: ForFSC
+              # User PIN
+              Pin: 98765432
+              Hash: SHA2
+              Security: 256
```

Comment:
If `Default: SW` in that case we do not need to set the `PKCS11` section, right?

Reply:
yes, but this is just an example. I'm following Fabric convetions.

Reply:
🙈